lib/data.js

```diff
@@ -0,0 +1,66 @@
+const data = [
```

Comment:
```suggestion
export const initialTodos = [
```

See my comment below.